### ISSUE TITLE
Comments count is off 

### ISSUE THREAD

**Issue by burntcookie90** [cheeaun/node-hnapi#18]
story `10963568` from `/news` shows 187 comments but hitting `item/10963568` shows only 9 for `comments_count`

see: https://whispering-fortress-7282.herokuapp.com/news
and: https://whispering-fortress-7282.herokuapp.com/item/10963568

**Comment by cheeaun** [comment]
So let me explain this; the cause of the bug is mainly from Firebase HN API *sometimes* returning 401 (Permission denied) responses for some of the comments. For the past few changes that I've committed, I purposely set a timeout for *every single* request to Firebase. If it's too slow, due to too many comments, it'll just silently fail. If it's 401, due to Firebase's weird bug, it'll also silently fail.

Before this solution, the *whole request* on the API (this one, not HN) fails, so I decided that it's still better to return *some* comments rather than *none*. Thus, explains why the comment count is wrong.

I'm not sure how to fix this, but hoping that it'll be fixed on Firebase side. Probably this one: https://github.com/HackerNews/API/pull/6

**Comment by burntcookie90** [comment]
Yeah I figured it was something similar, with the lack of updates on the hacker news firebase side, does it make sense to *not* use a scraper?
